src/app/pages/news/9.3/02-config-files.md

```diff
@@ -1,19 +1,8 @@
 ## Config files {#project93}
 
```

Comment:
give us a screenshot of the new sexy config files: e.g. showing partially the new project tree ...
plus an standard editor + the same resource opened in a split view using the .yaml editor?